augly/image/transforms.py

```diff
@@ -211,7 +211,35 @@ def apply_transform(
         @returns: Augmented PIL Image
         """
         return F.brightness(image, factor=self.factor, metadata=metadata)
-
+   
+   
+class Skew(BaseTransform):
```

Comment:
please insert this class in alphabetical order with respect to other classes in this file

Reply:
Done !!